ledger/sandbox/src/main/resources/db/migration/h2database/V13__Stable_offsets.sql

```diff
@@ -0,0 +1,25 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V13: Stable Offsets
+--
+-- Stable offsets are stored as varchar and can be sorted lexicographically.
+---------------------------------------------------------------------------------------------------
+ALTER TABLE parameters ALTER COLUMN ledger_end DROP NOT NULL;
+
+set referential_integrity false;
```

Comment:
I find it hard to believe that this actually works, but I guess you've tested it?

Reply:
For H2 we don't have pre-existing data and on an empty database it works.